packages/address-consts/src/index.ts

```diff
@@ -74,15 +74,9 @@ export interface Country {
 
 export interface ResponseError {
   errors: {
-    locations: {
-      column: number;
-      line: number;
-    }[];
     message: string;
-    problems: {
-      explanation: string;
-    }[];
-    value: any;
```

Comment:
What about `value: any;`?

Reply:
If `value` is being removed completely from this interface I believe that would constitute a breaking change. So a `2.0.0` release.

Reply:
The type of `locations` also change, so it's also a breaking change, and `problems` is also gone

Reply:
> What about value: any;?

Remove it because its not part of the GraphQL specs and wondering how did it even get there in the first place. Same as `problems` 

> The type of locations also changes
Fixed. My mistake. 

> constitute a breaking change. So a 2.0.0 release.
👍